zstacklib/zstacklib/utils/lvm.py

```
@@ -1716,6 +1716,11 @@ def get_lv_attr(lv_path, *attr):
     return o["report"][0]["lv"][0]
 
 
+def get_lv_lockargs(lv_path):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)